src/main/index.ts

```diff
@@ -35,6 +35,25 @@ protocol.registerSchemesAsPrivileged([
   {
     scheme: 'webxdc',
     privileges: {
+      // This gives apps access to APIs such as
+      // - Web Cryptography
+      // - Clipboard
+      // - Web Share
+      // , also see https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts/features_restricted_to_secure_contexts
+      //
+      // For reference:
+      // - https://support.delta.chat/t/allow-access-to-camera-geolocation-other-web-apis/2446?u=wofwca
+      //
+      // Note that APIs requiring explicit user permission (such as camera)
+      // still don't work, see
+      // https://github.com/deltachat/deltachat-desktop/blob/455a4d01501ed82f9d8e0a36064ffbc3981722ee/src/main/deltachat/webxdc.ts#L457-L473
+      //
+      // In terms of `isSecureContext`, webxdc apps are similar to files,
```

Comment:
files get less rights than https in reality, at least that's what electron docs says when I understood it correctly

Reply:
Do you have a link?

Reply:
> A standard scheme adheres to what RFC 3986 calls [generic URI syntax](https://tools.ietf.org/html/rfc3986#section-3). For example http and https are standard schemes, while file is not.

nvm seems like that was about the `standard` property.

still in browser the file scheme is heavily restricted at least in my experience.

Reply:
Can you give an example? Is it about `isSecureContext` or something else?

Reply:
if `isSecureContext` is true when you open the html file directly in the browser(no local server), then keep the comment about "file", if not remove "file" from the comment, after that we can merge it.

Reply:
Yes, it's true, at least for mainstream browsers.